### PR TITLE
added my name to readme file

### PR DESCRIPTION
**Pull request by senko-tail**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,4 @@ npm run web
 
 ## Authors
 Sandra Liljeqvist ([sanlil](https://github.com/sanlil))  
+Thomas Siu ([tsiu](https://github.com/senko-tail))
```
Comment (sanlil) [inline README.md:24]: Please add two spaces at the end of the line. This creates a new line in markdown. See example [here](https://gist.github.com/shaunlebron/746476e6e7a4d698b373)

**Comment by sanlil** [comment]
I see that your commit message is missing your GitHub profile picture. This is most likely since you haven't set your email address (the same as you use for GitHub) in your local Git config like [this](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address#setting-your-commit-email-address-in-git). Similarly you can also set your name (if you want to) like [this](https://docs.github.com/en/free-pro-team@latest/github/using-git/setting-your-username-in-git#setting-your-git-username-for-every-repository-on-your-computer).

File: README.md (hunk 1 above)
Comment (senko-tail) [reply README.md]: fixed

**Review by sanlil — APPROVED** [review]
Great! 🌟

**Pull request merged**